sdk/eventhub/event-hubs/README.md

```diff
@@ -117,69 +116,53 @@ main();
 Events may be added to the `EventDataBatch` using the [tryAdd](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventdatabatch.html#tryadd)
 method until the maximum batch size limit in bytes has been reached.
 
 ```javascript
 const { EventHubClient } = require("@azure/event-hubs");
 
 async function main() {
-  const client = new EventHubClient("connectionString", "eventHubName");
-  const producer = client.createProducer();
+  const producer = new EventHubProducerClient("connectionString", "eventHubName");
 
   const eventDataBatch = await producer.createBatch();
+  
   let wasAdded = eventDataBatch.tryAdd({ body: "my-event-body" });
   wasAdded = eventDataBatch.tryAdd({ body: "my-event-body-2" });
-  await producer.send(eventDataBatch);
+
+  await producer.sendBatch(eventDataBatch);
 
   await producer.close();
-  await client.close();
 }
 
 main();
 ```
 
 The [Inspect an Event Hub](#inspect-an-event-hub) example shows how to get the list of partition ids should you wish to specify one for a producer.
 
-The `createProducer` method takes an optional parameter of type [EventHubProducerOptions](https://azure.github.io/azure-sdk-for-js/event-hubs/interfaces/eventhubproduceroptions.html) which you can use to specify the retry options and partition id for the send operation.
+The `EventHubProducerClient` constructor takes an optional parameter of type `EventHubClientOptions` which you can use to specify options like number of retries.
 
-The `send` method takes an optional parameter of type [SendOptions](https://azure.github.io/azure-sdk-for-js/event-hubs/interfaces/sendoptions.html) which you can use to specify `abortSignal` to cancel current operation.
-You can also specify `partitionKey` if you did not specify a partition id when creating the producer.
+The `sendBatch` method takes an optional parameter of type `SendBatchOptions` which you can use to specify `abortSignal` to cancel current operation.
 All events that use the same partition key will be sent to the same partition.
 
 **Note**: When working with Azure Stream Analytics, the body of the event being sent should be a JSON object as well.
 For example: `body: { "message": "Hello World" }`
 
 ### Consume events from an Event Hub partition
 
-To consume events from a single Event Hub partition in a consumer group, create an `EventHubConsumerClient` for that partition and consumer group combination. You will need to provide a position in the event stream from where to begin receiving events; in our example, we will read new events as they are published.
+To consume events from a single Event Hub partition in a consumer group, create an [EventHubConsumerClient](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventhubconsumerclient.html) for that partition and consumer group combination. 
```

Comment:
> for that partition and consumer group combination

But we are not passing either to `EventHubConsumerClient`....

Reply:
Instead of showcasing consuming events from a single partition, lets have only examples for the multi partition scenario. This translates to 2 sections
- Consume events from all partitions
- Consume events from all partitions with balanced load.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/5964/commits/2c47d67e3c7402e11860d09c0caea074547dc1be